starter-code/index.html

```diff
@@ -84,7 +84,8 @@
     <script src="/scripts/article.js"></script>
```

Comment:
You need to remove all of the front slashes from the script tags to get your code working. This was a starter code error, nothing on your part. This has nothing to do with your grade, but it may help you debug.